.ci.yaml

```diff
@@ -4964,6 +4964,8 @@ targets:
       task_name: flutter_packaging
       tags: >
         ["framework", "hostonly", "shard", "mac"]
+    dimensions:
+      cpu: "arm64"
```

Comment:
I'll defer final approval. This file is confusing however, I see arm64 being specified under `cpu: arm64` https://github.com/flutter/flutter/blob/master/.ci.yaml#L90 and `properties.tags: ["arm64"]` https://github.com/flutter/flutter/blob/master/.ci.yaml#L3331, and then here under `dimensions.cpu: arm64`.

Reply:
Umm that's true, if it is specified in the general properties it should have been inherited

Reply:
To keep the dart internal builds simple they do not implement the logic post-process the ci.yaml file. They only see the specific target information.

Reply:
By the way we have an open bug to simplify properties and dimensions across flutter-infra, ci.yaml and cocoon. Turns out we have different implementations on those 3 places.

Reply:
Thanks for explaining! Didn't know dart internal uses a different parser for yaml